src/pepr/policies/security.ts

```diff
@@ -292,7 +348,11 @@ When(a.Pod)
 When(a.Pod)
   .IsCreatedOrUpdated()
   .Mutate(request => {
-    if (exemptDropAllCapabilities(request)) {
+    if (isExempt(Policy.DropAllCapabilities, request)) {
```

Comment:
I like the idea of this annotation to track exemptions--but think we could probably generalize this into a common function such that we could do `.Mutate(markExemption)` or something like that and let generics earn their pay.